apps/client-asset-sg/src/app/components/disclaimer-dialog/disclaimer-dialog.component.html

```diff
@@ -0,0 +1,16 @@
+<div>
+  <div class="header">
+    <h2 translate>disclaimer.title</h2>
+    <asset-sg-language-selector></asset-sg-language-selector>
+  </div>
+  <mat-divider></mat-divider>
+  <div mat-dialog-content>
+    <h1 translate>disclaimer.subtitle</h1>
+    <p [innerHTML]="'disclaimer.content' | translate | newlineToBr"></p>
+    <a *rxLet="currentLang$; let lang" [href]="lang" target="_blank" rel="noopener noreferrer">{{ lang }}</a>
```

Comment:
`noreferrer` is debatable here (it's more than okay to let the swissgeol main page know that we send users to it). I would also definitely add `external`, and maybe even `nofollow`.